src/main/ui/app/component/AddOnSearch.jsx

```diff
@@ -56,32 +56,45 @@ export default class AddOnSearch extends Component {
     }
 
     render() {
-        return (
-                <div>
-                    Type:
-                    <select onChange={event => this.setType(event.target.value)}>
-                        <option value="">All Types</option>
-                        <option value="OMOD">Module (OMOD)</option>
-                        <option value="OWA">Open Web App (OWA)</option>
-                    </select>
 
-                    <br/>
+const title = (
+<span>Type: {this.state.type}</span>
+);
+        return (
+                <div className="row col-md-12 col-sm-12 col-xs-12 pushdown">
+                    <div>
+    <div className="input-group">
+    
+      <div className="input-group-btn">
+       
+ <DropdownButton title={title}  id="dropdown-size-small" onSelect={event => this.setType(event)}>
+        <MenuItem eventKey="">All Types</MenuItem>
+        <MenuItem eventKey="OMOD" >Module (OMOD)</MenuItem>
+        <MenuItem eventKey="OWA">Open Web App (OWA)</MenuItem>
+      </DropdownButton>
+      </div>
+  
+  
                     <DebounceInput
                             placeholder="Search..."
-                            className="search"
+                            className="form-control col-md-10 col-sm-10 col-xs-10"
```

Comment:
As a general comment, I thought that with bootstrap's grid system, if you're going to set the column size to be the same on different device sizes, you don't need to duplicate it. E.g. just doing `col-md-10` is enough without repeating it for sm and xs. (But I'm not 100% sure about that.)